test/specs/mounting-options/slots.spec.js

```diff
@@ -109,10 +109,13 @@ describeWithMountingMethods('options.slots', (mountingMethod) => {
       const wrapper5 = mountingMethod(ComponentWithSlots, { slots: { default: '1{{ foo }}2' }})
       expect(wrapper5.find('main').html()).to.equal('<main>1bar2</main>')
       wrapper5.trigger('keydown')
+      // expect(wrapper5.find('main').html()).to.equal('<main>1BAR2</main>')
```

Comment:
@eddyerburgh 
This line does not work.
Would you please check below links?

* https://github.com/vuejs/vue-test-utils/pull/274#issuecomment-352262141

* https://github.com/vuejs/vue-test-utils/pull/274/commits/3b99090a6917b7861096b2903fc9ec7811607464#diff-dbfd1670fb2dd50be051adb6643c8c24